src/components/breadcrumb/breadcrumb.html

```diff
@@ -1,4 +1,4 @@
-<div class="bx--breadcrumb bx--breadcrumb--no-trailing-slash">
+<nav class="bx--breadcrumb bx--breadcrumb--no-trailing-slash" aria-label="breadcrumb">
```

Comment:
Pretty sure the nav is needed because this component is used for navigation between pages. Unsure of the `aria-label`